source/github-helpers/selectors.ts

```diff
@@ -197,6 +197,7 @@ const authorLinks = [
 	'h3 a[data-testid="issue-body-header-author"]', // The first issue comment
 	'.react-issue-comment a[data-testid="avatar-link"]',
 	'[data-testid="comment-header"] a[data-testid="avatar-link"]', // React commit view
+	'.react-issue-body a[data-testid="issue-body-header-author"]', // React issue view first comment
```

Comment:
#8021 doesn't work now, GitHub has updated the `test-id` here.

Reply:
If selectors are no longer matching where they used to be, we should drop them. The view is still in beta so the selector will not appear in GHE.